src/DIRAC/WorkloadManagementSystem/Client/PilotLoggingPlugins/PilotLoggingPlugin.py

```diff
@@ -0,0 +1,52 @@
+"""
+Basic Pilot logging plugin. Just log messages.
+"""
+from DIRAC import S_OK, S_ERROR, gLogger
+
+sLog = gLogger.getSubLogger(__name__)
+
+
+class PilotLoggingPlugin:
+    """
+    This is a no-op fallback solution class, to be used when no plugin is defined for remote logging.
+    Any pilot logger plugin could inherit from this class to receive a set of no-op methods required by
+    :class:`TornadoPilotLoggingHandler` and only overwrite needed methods.
+    """
+
+    def __init__(self):
+        sLog.warn("PilotLoggingPlugin is being used. It only logs locally at a debug level.")
+
+    def sendMessage(self, message, pilotUUID, vo):
+        """
+        Dummy sendMessage method.
+
+        :param str message: text to log in json format
+        :param str pilotUUID: pilot id. Optimally it should be a pilot stamp if available, otherwise a generated UUID.
+        :param str vo: VO name of a pilot which sent the message.
+        :return: S_OK or S_ERROR
+        :rtype: dict
+        """
+        sLog.debug(message)
+        return S_OK("Message sent")
```

Comment:
Since this is a no-op, shouldn't this be

```suggestion
        return S_ERROR()
```
?

Reply:
Contains abstract methods only.